tests/storage/snapshots/utils.py

```
@@ -44,3 +60,58 @@ def fail_to_create_snapshot_no_permissions(snapshot_name, namespace, vm_name, cl
+                client=client,
+                source_dict=construct_datavolume_source_dict(source="blank"),
+                size=BLANK_DV_SIZE,
+                storage_class=storage_class_name,
+                api_name="storage",
+            )
+            blank_dv.to_dict()
+            blank_dv.res["metadata"].pop("namespace", None)
+            add_dv_to_vm(vm=vm, template_dv=blank_dv.res)
+        running_vm(vm=vm)
+    except Exception as error:
+        LOGGER.error(f"Failed to set up VM {vm_name}, cleaning up: {error}")
```

Ruff (0.16.2)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)